src/components/FireBase/Storage/fileManipulation.js

```diff
@@ -0,0 +1,27 @@
+import { Platform } from 'react-native';
+
+import FireBaseStorage from './api';
+
+const getFileLocalPath = (response) => {
+  const { path, uri } = response;
+  return Platform.OS === 'android' ? path : uri;
+};
+
+const createStorageReferenceToFile = (response) => {
+  const { fileName } = response;
+
+  // TODO: Move this to somewhere else
+  const id = Math.random()
+    .toString(36)
+    .slice(3);
+
+  return FireBaseStorage.ref(`images/${id}-${fileName}`);
+};
+
+const uploadFileToFireBase = (response) => {
+  const fileSource = getFileLocalPath(response);
+  const storageRef = createStorageReferenceToFile(response);
+  return storageRef.putFile(fileSource);
+};
+
+export default uploadFileToFireBase;
```

Comment:
function should be UploadFileToFirebase, and the file name should be either UploadFileToFirebase/FileManipulation

instead of setting a const as a function, we can just do:
`export default function UploadFileToFirebase (response) { ...`